### ISSUE TITLE
Applicant Timeline - Modal accessibility

### ISSUE THREAD

**Issue by gggrant** [GCTC-NTGC/TalentCloud#5161]
- Use H1 elements for the modal title.
- When the modal opens, focus should be on the H1.
- Fix keyboard trap (cannot cycle through modal contents with tab).
- Ensure all form elements are focusable.
- Check the date picker, possibly causing some of these issues.

**Comment by tristan-orourke** [comment]
Focus on the new Application and new profile pages.

**Comment by tristan-orourke** [comment]
Lets start by replacing the Experience dialogs with the ones built with the H2 dialog component, and then test again.
